snapcraft/how-to/interface-management.md

```diff
@@ -1,26 +1,21 @@
 [Interfaces](/t/35928) allow (or deny) access to a resource outside of a snap's confinement.
 
-Most users don't need to worry about interfaces. Snaps are designed for strong application isolation and safe interface connections are made automatically. 
+Most users don't need to worry about interfaces. Snaps are designed for strong application isolation and safe interface connections are made automatically.
 
 An interface is most commonly used to enable a snap to access sound playback or recording, your network, and your $HOME directory. But which interfaces a snap requires, and *provides*, is very much dependent on the type of snap and its own requirements.
 
 See [Supported interfaces](/t/supported-interfaces/7744) for a comprehensive list of interfaces and what kind of access they permit.
 
-<h2 id='heading--slots-plugs'>Plugs and slots</h2>
+## View all the snaps using an interface
 
-An interface provides a level of access to resources, such as audio playback, as defined by a *slot*. One or more snaps can access this resource by connecting a corresponding *plug* to the slot.
+Run the command:
 
-In other words, the slot is the provider of the resource while the plug is the consumer, and a slot can support multiple plug connections.
-
-![How an interfaces uses a plug and a slot](https://assets.ubuntu.com/v1/59c290a8-snapd-interfaces.png) 
-
-In the output to `snap connections vlc` (see above), every interface used by VLC is listed in the first column. The *Plug* and *Slot* columns then describe how each interface is connected.
-
-For instance, the `audio-playback` interface connects VLC's audio-playback plug to the system's audio-playback slot so you can hear the sound it produces.
+```bash
+snap interface <interface name>
 
-<h2 id='heading--listing'>Listing interfaces</h2>
+```
 
-You can see which snaps are using an interface with the `interface` command:
+The `interface` command shows the snaps using an interface. For example, for the audio-playback interface you'd see:
```

Comment:
I think there's a hint of ambiguity here in _shows the snaps using an interface_ which is easier to understand with the _which_.

```suggestion
The `interface` command shows which snaps are using an interface. For example, for the audio-playback interface you'd see:
```